### DISCUSSION TITLE
Can can I specify a fallback css property for legacy browsers?

### DISCUSSION THREAD

**Discussion by preziotte** [chakra-ui/chakra-ui#7993] [Q&A]
I'd like to redundantly set two `maxHeight` properties on a `Box`.   Like so:

```
<Box
   maxHeight="100dvh"
   maxHeight="100vh"
   ...
```

Since dvh only has [88% adoption](https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic), I want to also set the vh value as a fallback for browsers that don't support it.  Is there a way to do this?

**Comment by DubMan21** [comment]
Did you find a solution @preziotte ?

**Reply by preziotte** [reply]
I have not 😢

**Reply by raymondjacobson** [reply]
You could try: `min(100dvh, 100vh)`
